content/docs/guides/branching-githooks.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Automate branching with Githooks
+subtitle: Learn how you can automatically create and delete database branches in Neon using Githooks
+enableTableOfContents: true
+isDraft: true
+---
+
+This guide leads you through creating a Githook script that automates the creation of Neon branches every time a new Git branch is created. This is possible because Neon provides an API to manage your projects, branches, and most other operations supported by the Neon Console. You can learn more about the Neon API [here](https://neon-console-docs-link.com).
```

Comment:
missed the link? :)

Reply:
mean to remove this content from the PR, as per the comment below.